htsget.md

```diff
@@ -37,31 +37,53 @@ Requests to the retrieval API endpoint may be authenticated by means of an OAuth
 
 ## Errors
 
-Non-successful invocations of the API return an HTTP error code, and the response body contains a JSON object (content-type application/json) with the following structure:
+The server MUST respond with an appropriate HTTP status code (4xx or 5xx) when
+an error condition is detected.  In the case of transient server errors, 
+(e.g., 503 and other 5xx status codes), the client SHOULD implement appropriate
+retry logic as specified in [RFC2616&sect;10.5](https://tools.ietf.org/html/rfc2616#section-10.5).
 
-```json
-{
-   "error": {
-       "type": "NotFound",
-       "message": "No such accession"
-   }
-}
-```
+For errors that are specific to the `htsget` protocol, the response body SHOULD be a JSON object
+(content-type `application/json`) providing machine-readable information about the nature 
+of the error, along with a human-readable description. The structure of this JSON object is 
+described as follows.
+
+### Error Response JSON fields
+
+<table>
+<tr markdown="block"><td>
+`error`  
+_string_
+</td><td>
+The type of error. This SHOULD be chosen from the list below.
+</td></tr>
+<tr markdown="block"><td>
+`message`  
+_string_
+</td><td>
+A message specific to the error providing information on how to debug the problem. Clients 
+MAY display this message to the user.
+</td></tr>
+</table>
```

Comment:
I agree it would be better if we could just use Markdown, but I'm going to take @jmarshall's word that this is hard to do if we're using formatting within tables. We should follow this up with another issue/PR though once we've agreed on the actual content for this PR.